### DISCUSSION TITLE
Request a translation of a Chinese document.

### DISCUSSION THREAD

**Discussion by DeepExpBin** [CaiJimmy/hugo-theme-stack#777] [Ideas]
Hello author, I like the theme you made very much, so I would like to request to participate in the project and translate a Chinese document for Chinese users to use.

**Comment by CaiJimmy** [comment]
Thanks for your interest.

We used to have Chinese documentation, but it was outdated and I didn't have enough resources and time to maintain it. Hence I removed it after switching to Vitepress: https://github.com/CaiJimmy/hugo-theme-stack/discussions/688

VitePress does not seem to have official i18n support yet. So it might not be easy to implement it.
